Source/reactions/Castro_react.cpp

```diff
@@ -11,6 +11,114 @@ using namespace amrex;
 
 #ifndef TRUE_SDC
 
+advance_status
+Castro::do_old_reactions (Real time, Real dt)
+{
+    amrex::ignore_unused(time);
+    amrex::ignore_unused(dt);
+
+    bool burn_success = true;
+
+#ifndef SIMPLIFIED_SDC
+    MultiFab& R_old = get_old_data(Reactions_Type);
```

Comment:
isn't it the case that `do_old_reactions` should never be called for simplified-SDC?

Reply:
The contents of the function are basically ifdef'ed out for that case. I just preferred to have the ifdef inside the function rather than in Castro_advance